src/api/apiConfig.js

```diff
@@ -1,28 +1,11 @@
-import { config } from 'd2'
-
-export const getBaseURL = () => {
-    let baseURL
-
-    let apiUrls = {
-        development: "http://localhost:8080/api",
-        production: config.baseUrl
-    }   
-    
-    if(window.location.hostname === "localhost") {
-        baseURL = apiUrls.development
-    } else {
-        baseURL = apiUrls.production
-    }
-    return baseURL
-}
-
 export const options = {
     headers: {
         'X-Requested-With': 'XMLHttpRequest',
         'Content-Type': 'application/json',
         Accept: 'application/json',
     },
     credentials: "include",
+    timeout: '60000'
```

Comment:
I noticed it was frequently timing out when "publishing to DHIS2", so I think there should be a higher timeout than default (which is 10s).